CLI.md

```diff
@@ -869,7 +869,7 @@ Start a Local Linera Network
 
 * `--testing-prng-seed <TESTING_PRNG_SEED>` — Force this wallet to generate keys using a PRNG and a given seed. USE FOR TESTING ONLY
 * `--path <PATH>` — Run with a specific path where the wallet and validator input files are. If none, then a temporary directory is created
-* `--storage-config-namespace <STORAGE_CONFIG_NAMESPACE>` — Run with a specific storage. If none, then a linera-storage-service is spanned on a random free port
+* `--storage <STORAGE>` — Run with a specific storage. If none, then a linera-storage-service is spanned on a random free port
```

Comment:
Should this say "spawned"? Or maybe "started" or "run"?

Reply:
Corrected.